application/engine/services/autopilot_daemon.py

```
@@ -1168,3 +1174,74 @@ async def _maybe_generate_summaries(self, novel: Novel, completed_count: int) ->
+                novel_id=novel_id_str,
+                novel_title=novel.title,
+                next_chapter=next_chapter,
+                scene_type=scene_type,
+                prev_tension=prev_tension,
+                act_number=act_number,
+                base_prompt=base_prompt,
+            )
+            if result != base_prompt:
+                logger.info(f"[Hermes] novel={novel_id_str}, prompt enhanced with skills")
+            return result
+        except Exception as e:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)